rpc/chain/client.go

```diff
@@ -172,7 +172,9 @@ func isVMError(err error) bool {
 }
 
 func isRPSLimitError(err error) bool {
-	return strings.Contains(err.Error(), "backoff_seconds")
+	return strings.Contains(err.Error(), "backoff_seconds") ||
```

Comment:
these strings are provider-specific, we probably want a tidier way of handling them. Can be done in a separate task though